src/components/tracked-controls.js

```diff
@@ -21,6 +21,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
     rotationOffset: {default: 0},
+    hand: {type: 'string'},
```

Comment:
@dmarcos specifically wanted to remove hand from tracked-controls, so was surprised to see you add it back in.  I had actually renamed it to defaultHand before @dmarcos asked to remove it entirely and just use right DEFAULT_HANDEDNESS constant.  so I am not sure we want this, per that prior discussion?